docs/resources/Soundboard.md

```diff
@@ -0,0 +1,90 @@
+
+There is a set of [default sounds](#DOCS_RESOURCES_SOUNDBOARD/list-soundboard-default-sounds) available to all users. Soundboard sounds can also be [created in a guild](#DOCS_RESOURCES_SOUNDBOARD/create-guild-soundboard-sound); users will be able to use the sounds in the guild, and Nitro subscribers can use them in all guilds.
+
+Soundboard sounds in a set of guilds can be retreived over the Gateway using [Request Soundboard Sounds](#DOCS_TOPICS_GATEWAY_EVENTS/request-soundboard-sounds).
+
+### Soundboard Sound Object
+
+###### Soundboard Sound Structure
+
+| Field         | Type                                            | Description                                                                                  |
+| ------------- | ----------------------------------------------- | -------------------------------------------------------------------------------------------- |
+| name          | string                                          | the name of this sound                                                                       |
+| sound_id      | snowflake                                       | the id of this sound                                                                         |
+| id?           | snowflake                                       | the id of this sound                                                                         |
+| volume        | double                                          | the volume of this sound, from 0 to 1                                                        |
+| emoji_id      | ?snowflake                                      | the id of this sound's custom emoji                                                          |
+| emoji_name    | ?string                                         | the unicode character of this sound's standard emoji                                         |
+| override_path | ?string                                         | the filename of this sound (for default sounds)                                              |
+| guild_id?     | snowflake                                       | the id of the guild this sound is in                                                         |
+| user_id       | snowflake                                       | the id of the user who created this sound                                                    |
+| available?    | boolean                                         | whether this sound can be used (for guild sounds), may be false due to loss of Server Boosts |
+| user?         | [user](#DOCS_RESOURCES_USER/user-object) object | the user who created this sound                                                              |
+
+### Sound Files
```

Comment:
maybe that could be integrated into the reference section somehow, since that's the only place where CDN endpoints are currently listed...